app/js/subProblem/subProblemController.js

```diff
@@ -147,6 +136,38 @@ define(['lodash', 'clipboard', 'angular'], function(_, Clipboard) {
         }
       });
     }
+
+    function deleteSubproblem() {
+      $modal.open({
+        templateUrl: './deleteSubproblem.html',
+        controller: 'DeleteSubproblemController',
+        resolve: {
+          subproblem: function() {
+            return $scope.subProblem;
+          },
+          callback: function() {
+            return function() {
+              SubProblemResource.delete($stateParams).$promise.then(function() {
+                var otherSubproblem = _.reject($scope.subProblems, ['id', $scope.subProblem.id])[0];
```

Comment:
othersubproblemS I think? In which case it's not consistent with usage of subProblemChanged (which expects a single problem)

Reply:
[0] at the end

Reply:
doh